### ISSUE TITLE
Make a Weather App

### ISSUE THREAD

**Issue by savneetkaur03** [GDSC-CEC/HacktoberFest-2022-Android#9]
Make a weather app. There is already a folder named "Weather app" inside the easy folder, make your project inside that folder only.
Also, make sure to follow the guidelines while making PR for the issue.

**Comment by shahGnp** [comment]
Can I work on this?

**Comment by savneetkaur03** [comment]
Yes. I am assigning this task to you. Kindly try to submit the project in two-three days.
